{"name": "Test04: Filtering Users", "status": "failed", "statusDetails": {"message": "AssertionError: Number of elements in list: 2does not match expected :8", "trace": "self = <test_cases.test_web.Test_Web object at 0x000001AD9938D040>, search_value = 'gmail', expected_users = '8'\n\n    @allure.title(\"Test04: Filtering Users\")\n    @allure.description(\"this test filters users\")\n    @pytest.mark.parametrize('search_value, expected_users', web_flows.testdata)\n    def test_csv(self, search_value, expected_users):\n        WebFlows.open_user()\n        WebFlows.search_user(search_value)\n>       WebFlows.verify_number_of_users(int(expected_users))\n\ntest_web.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\workflows\\web_flows.py:67: in verify_number_of_users\n    Verifications.verify_number_of_elements(page.web_server_admin_page.get_users_list(), number)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nelems = [<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6580>, <selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6E50>]\nsize = 8\n\n    @staticmethod\n    @allure.step(\"verify number of elements in list\")\n    def verify_number_of_elements(elems, size):\n>       assert len(elems) == size, 'Number of elements in list: ' + str(len(elems)) + 'does not match expected :' + str(size)\nE       AssertionError: Number of elements in list: 2does not match expected :8\n\n..\\extensions\\verifications.py:42: AssertionError"}, "description": "this test filters users", "steps": [{"name": "go to users flow", "status": "passed", "steps": [{"name": "mouse hover", "status": "passed", "parameters": [{"name": "elem1", "value": "<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD99475190>"}, {"name": "elem2", "value": "<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD99475CD0>"}], "start": 1683803232196, "stop": 1683803232773}], "start": 1683803232163, "stop": 1683803232773}, {"name": "search user from users table flow", "status": "passed", "steps": [{"name": "clear text field in element", "status": "passed", "parameters": [{"name": "elem", "value": "<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6DC0>"}], "start": 1683803232815, "stop": 1683803232881}, {"name": "updating text", "status": "passed", "parameters": [{"name": "elem", "value": "<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6DC0>"}, {"name": "value", "value": "'gmail'"}], "start": 1683803232888, "stop": 1683803233000}], "parameters": [{"name": "search_value", "value": "'gmail'"}], "start": 1683803232774, "stop": 1683803233000}, {"name": "verify number of users in table flow", "status": "failed", "statusDetails": {"message": "AssertionError: Number of elements in list: 2does not match expected :8\n", "trace": "  File \"C:\\Automation\\test_Automation_final_project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Automation\\test_Automation_final_project\\workflows\\web_flows.py\", line 67, in verify_number_of_users\n    Verifications.verify_number_of_elements(page.web_server_admin_page.get_users_list(), number)\n  File \"C:\\Automation\\test_Automation_final_project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Automation\\test_Automation_final_project\\extensions\\verifications.py\", line 42, in verify_number_of_elements\n    assert len(elems) == size, 'Number of elements in list: ' + str(len(elems)) + 'does not match expected :' + str(size)\n"}, "steps": [{"name": "verify number of elements in list", "status": "failed", "statusDetails": {"message": "AssertionError: Number of elements in list: 2does not match expected :8\n", "trace": "  File \"C:\\Automation\\test_Automation_final_project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Automation\\test_Automation_final_project\\extensions\\verifications.py\", line 42, in verify_number_of_elements\n    assert len(elems) == size, 'Number of elements in list: ' + str(len(elems)) + 'does not match expected :' + str(size)\n"}, "parameters": [{"name": "elems", "value": "[<selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6580>, <selenium.webdriver.support.event_firing_webdriver.EventFiringWebElement object at 0x000001AD993F6E50>]"}, {"name": "size", "value": "8"}], "start": 1683803233521, "stop": 1683803233521}], "parameters": [{"name": "number", "value": "8"}], "start": 1683803233000, "stop": 1683803233521}], "attachments": [{"source": "36043376-fe87-4651-8ab7-b8700412dd01-attachment.png", "type": "image/png"}], "parameters": [{"name": "search_value", "value": "'gmail'"}, {"name": "expected_users", "value": "'8'"}], "start": 1683803232163, "stop": 1683803233522, "uuid": "4b93cff2-b508-44a9-9e87-075a0a9cf7f4", "historyId": "666264e5c44bbf4e1b94e5e8ae0c3dfe", "testCaseId": "28d40b7d3b332b3990bd9c25b6981ed4", "fullName": "test_web.Test_Web#test_csv", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_web_driver')"}, {"name": "suite", "value": "test_web"}, {"name": "subSuite", "value": "Test_Web"}, {"name": "host", "value": "LAPTOP-KLQ6N0KI"}, {"name": "thread", "value": "5812-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_web"}]}